### PR TITLE
Replace rollup with webpack

### PR DESCRIPTION
**Pull request by bfred-it**
I was growing tired of rollup's over-modularity, so I thought of just using webpack.

Pros:

- shows each bundle's size and can show which module is too heavy
- faster builds: 500ms vs 1000ms.
  In watch mode the difference is higher and with sourcemaps enable the difference is 6x.
- fewer bugs
- ironically shorter config

Cons:

- bundle 10KB heavier (129KB vs 119KB)
- requires an extra `babel` step on `release` to strip comments (otherwise it'd be 149KB)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 {
 	"scripts": {
 		"test": "xo && ava && npm run build",
-		"build": "rollup -c",
-		"watch": "run-p --continue-on-error --silent \"build -- --watch\"",
-		"watch:sourcemap": "cross-env SOURCEMAP=true npm run watch --silent",
+		"build": "webpack",
+		"watch": "webpack --watch",
+		"watch:sourcemap": "webpack --watch --devtool eval-cheap-module-source-map",
+		"reduce-size": "babel --out-dir . --no-comments extension/*.js",
```
Comment (sindresorhus) [inline package.json]: Can we have this in the webpack config instead?

Comment (sindresorhus) [inline package.json]: Oh, I see you originally had that, why did you revert it?

https://github.com/sindresorhus/refined-github/pull/577/commits/bd8b1a6aa9cb453b3959f881f3f822ce9f6cd916

Comment (bfred-it) [inline package.json]: It just wasn't applied :(

It might be a babel or babel-loader bug. That'd be its right place

**Comment by sindresorhus** [comment]
Discovered a weird problem. On master the "Hide other users starring/forking your repos" setting is checked for me, but when running this PR it's not.

**Comment by bfred-it** [comment]
My bad. Part of an incomplete f393565. Fix coming

File: package.json (hunk 1 above)
Comment (arkon) [inline package.json]: Why not use `webpack.optimize.UglifyJsPlugin` to strip comments?

Comment (bfred-it) [inline package.json]: No ES6 support :(

Comment (arkon) [inline package.json]: Ah true. That's weird how the options weren't working though. Maybe `babelrc: false` needs to be explicitely specified despite not existing anyway? Might be worth considering using [babili](https://github.com/babel/babili) too.

**Pull request merged**